includes/UserAdd.php

```diff
@@ -115,6 +116,12 @@
 		echo "<h1 style ='color:red'>Error updating database information, Try another username because same user name already exists</h1>";
 		die();
 	}
+	
+	else
+	{
+		$flag = 1 ;
+	}
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->